jump.py

```diff
@@ -0,0 +1,206 @@
+import cv2
+import mediapipe as mp
+import numpy as np
+import time 
+import pyautogui
+
+mp_drawing = mp.solutions.drawing_utils
+mp_pose = mp.solutions.pose
+
+
+### CONSTANTS 
+video_height = 720
+video_width = 720
+
+
+#Counter variables
+counterDuck = 0 
+counterJump = 0
+counterSwing = 0
+stageDuck = None
+stageJump = None
+stageSwing = None
+
+#action status
+inJump = False
+inDuck = False
+inSwing = False
+
+#angle
+left_angle = None
+right_angle = None
+
+#other variables
+previousHeight = None
+
+def calculate_angle(a,b,c):
+    a = np.array(a) # First
+    b = np.array(b) # Mid
+    c = np.array(c) # End
```

Comment:
what does first, mid, end refer to exactly? I think the comment should be a bit clearer